linode/manager
# M3-5946: Cypress 10 clean up - refactor plugin file

## Description 📝

**What does this PR do?**
This refactors our Cypress plugin code by removing `plugins/index.js` and moving that functionality to `cypress.config.ts` as described in [Cypress's Migration Guide](https://docs.cypress.io/guides/references/migration-guide#Plugins-File-Removed). It also removes some plugin logic that isn't needed, and upgrades the `dotenv` dev dependency for TypeScript support.

## How to test 🧪

**What are the steps to reproduce the issue or verify the changes?**
```bash
yarn cy:run
```

You don't necessarily have to run the entire set of tests; the plugin code is responsible for loading the `.env` file, so if there is a problem you will quickly observe either an error loading the file, or test failures due to API 403 responses.
